adal/src/main/java/com/microsoft/aad/adal/Oauth2.java

```diff
@@ -634,6 +637,10 @@ private AuthenticationResult processTokenResponse(HttpWebResponse webResponse, f
                     httpEvent.setRequestIdHeader(listOfHeaders.get(0));
                 }
             }
+
+            if (null != webResponse.getResponseHeaders().get(X_MS_CLITELEM) && !webResponse.getResponseHeaders().get(X_MS_CLITELEM).isEmpty()) {
+                httpEvent.setXMsCliTelemData(webResponse.getResponseHeaders().get(X_MS_CLITELEM).get(0));
```

Comment:
will it add any value if adding logs here?

Reply:
There was originally some logging around processing this header (albeit in a different location, although with the same effect) - CR feedback from @addev-ashish was to remove it to cut down noise. I'm ok with this in either case